browser/src/react-redux/App/User.reducer.js

```diff
@@ -1,5 +1,10 @@
 const initialState = null;
 
 export default (state = initialState, action) => {
-    return state;
+    switch (action.type) {
+        case 'NEW_USER_SUBMITTED':
+            return { ...state, ...action.payload }
```

Comment:
Here's my thoughts on the structure of the store. We probably just want to store the user here. Let's assume our signup component reroutes the user back to the home page if they're logged on. (Let's add that feature in the component). Now, we know that if this action is called, we want to store a new user object. We don't need the previous state - we can just do `return { ...action.user}`. Notice that I changed `.payload` to `.user` as I think that will be clearer of what we're receiving _from_ the payload.